<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infoBoard</class>
 <widget class="QWidget" name="infoBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="proNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Magnus Carlsen</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="overviewLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Overview:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="majorWinsLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Major Wins:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="peakRatingLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>• Peak Rating: 2882</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>• Title: Grandmaster (GM) </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="birthLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>• Born: 30 November, 1990 </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="genderLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>• Gender: Male</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="nationalityLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>• Nationality: Norwegian</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="major1Label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>• 5x World Chess Champion</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="major2Label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>• Triple Crown Winner</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="major3Label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>• Highest ever FIDE Rating</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>521</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
    </font>
   </property>
   <property name="text">
    <string>Magnus Carlsen is a Norwegian chess grandmaster and one of the greatest players in history. He became a grandmaster at 13, quickly challenging the world's best with his sharp intuition. His deep understanding of chess and consistent top-level performance have made him a dominant force since 2010, setting numerous records in the process.

Carlsen's playstyle is marked by versatility and adaptability, thriving in any position. He excels in all phases of the game—openings, middlegames, and endgames—and is known for squeezing wins from seemingly drawn positions with exceptional precision. His legendary endgame skills, combined with an uncanny ability to maintain focus under pressure, make him incredibly difficult to beat, often outlasting opponents in long, grueling battles.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="backLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>168</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="forwardLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>168</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
